integration-tests/smoke/ocr2_test.go

```diff
@@ -146,8 +146,12 @@ var _ = Describe("StarkNET OCR suite @ocr", func() {
 			sum`
 
 			observationSource := `
-			sum [type="sum" values=<[900000000]>]
-			sum
+			pos [type="sum" values=<[900000000]>]
+			neg [type="sum" values=<[-900000000]>]
+			pick [type="any"]
+			
+			pos -> pick
+			neg -> pick
```

Comment:
this introduces some non-deterministic behavior because the job run will randomly choose a positive or negative value. the test will run will continue until both a positive and negative value have been seen onchain

Reply:
![image](https://user-images.githubusercontent.com/1372918/186817081-86fb4d35-92cf-4d76-8df0-bb7f33ea8317.png)

Reply:
lol fixed this - with an EA and manually changing the returned value after a few successful rounds
`900000000 => -900000000`